LabGui.py

```diff
@@ -1523,11 +1523,46 @@ def relaunch(self, force=False, **kwargs):
         #self.exit(self.EXIT_CODE_REBOOT)
     #    relaunch_LabGui()
 
+################ ENABLE ERROR HANDLING #######################
+ex = None
+app = None
+
+sys._excepthook = sys.excepthook
+def error_message(value):
+    global ex
+    msg = QtGui.QMessageBox.critical(ex,
```

Comment:
if you move this to the next line it can be indented more to the right (only aesthetic)